src/kubetop/_script.py

```diff
@@ -36,7 +37,8 @@ def current_context(config_path):
 
 class KubetopOptions(Options):
     optParameters = [
-        ("context", None, current_context(CONFIG), "The kubectl context to use."),
+        ("config", None, DEFAULT_CONFIG, "The path to the kubectl config to use."),
+        ("context", None, current_context(DEFAULT_CONFIG_FILE_PATH), "The kubectl context to use."),
```

Comment:
I wonder if this will be misleading.  `--help` will show the context that's active in `DEFAULT_CONFIG` but if you then pass `--config ...` with another path and don't pass `--context ...` kubetop will actually use whatever context is active in that alternate config file.  Since kubetop is read-only I don't think this can actually hurt anything but it might be confusing or surprising.

An alternative could be to remove the default value from the `KubetopOptions` definition entirely and compute it later (eg in `postOptions`) from the value of `config`.  This doesn't change any behavior except `--help` will no longer show the default value for `--context` which might be better than showing a maybe-wrong default.